src/assets/translations/en/main.json

```diff
@@ -2417,5 +2419,27 @@
       "title": "Lending",
       "body": "Borrow against your assets."
     }
+  },
+  "RFOX": {
```

Comment:
Same for all the copies here, not a blocker obviously but always better for globalization to reuse translations (even if it looks semantically odd to consume e.g `pools.waitingForConfirmation` for RFOX, to avoid them having to re-translate and have translations already available